composer.json

```diff
@@ -16,7 +16,7 @@
   },
   "require-dev": {
     "facebook/fbexpect": "^2.0",
-    "hhvm/hhast": "^4.0",
```

Comment:
Sorry, just meant that the new release had to actually be released; this should be automatically picked up based on hhast's own version requirements, now that it's happened.